client/my-sites/people/people-profile/index.jsx

```diff
@@ -199,14 +204,32 @@ class PeopleProfile extends React.PureComponent {
 			);
 		}
 
-		if ( ! roleBadge && ! superAdminBadge ) {
+		if ( isContractor ) {
+			contractorBadge = (
+				<Fragment>
+					<div className="people-profile__role-badge role-contractor">
+						{ translate( 'Contractor', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Contractor' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?